lab-jinho/index.js

```diff
@@ -0,0 +1,6 @@
+'use strict';
+
+const greet = require('./lib/greet.js');
+
+greet.sayHey();
```

Comment:
I notice that in greet.js the sayHey method is expecting to be passed a name so you will want to pass it an argument so it actually outputs 'hey <name>'.